.storybook/stories/wizard/wizard.stories.ts

```diff
@@ -61,6 +62,7 @@ const defaultParameters: Parameters = {
     clrWizardPreventDefaultNext: { defaultValue: false },
     clrWizardPreventDefaultCancel: { defaultValue: false },
     clrWizardPreventModalAnimation: { defaultValue: false },
+    clrWizardStepnavAriaLabel: { defaultValue: 'Step navigation', control: { type: 'text' } },
```

Comment:
1. I think `control: { type: 'text' }` can be removed since it can be inferred.
2. I think we should use `commonStringsDefault.wizardStepnavAriaLabel` here like in other stories so it's updated for us if the default value is changed.